helpers/report_hunter.py

```diff
@@ -115,10 +137,9 @@ def main(latest: bool = False):
             exome_output = 'Exome' if 'exome' in output_path else 'Familial'
             singleton_output = 'Singleton' if 'singleton' in output_path else 'Familial'
             try:
-                # pipeline runs don't have display_url
                 report_address = analysis['output'].replace(
-                    get_config()['storage']['default']['web'],
-                    get_config()['storage']['default']['web_url'],
+                    get_config(False)['storage'][cohort]['web'],
+                    get_config(False)['storage'][cohort]['web_url'],
```

Comment:
TIL `False` => don't print out the config. Now that is good to know about! :smile: